specs-jetpack-calypso/wp-jetpack-connect-spec.js

```diff
@@ -0,0 +1,121 @@
+/** @format */
+import test from 'selenium-webdriver/testing';
+import config from 'config';
+import assert from 'assert';
+
+import * as driverManager from '../lib/driver-manager';
+import * as driverHelper from '../lib/driver-helper';
+import * as dataHelper from '../lib/data-helper';
+import { By } from 'selenium-webdriver';
+
+import AddNewSitePage from '../lib/pages/add-new-site-page';
+import PickAPlanPage from '../lib/pages/signup/pick-a-plan-page';
+import SettingsPage from '../lib/pages/settings-page';
+import WporgCreatorPage from '../lib/pages/wporg-creator-page';
+import LoginFlow from '../lib/flows/login-flow';
+import SidebarComponent from '../lib/components/sidebar-component';
+
+const mochaTimeOut = config.get( 'mochaTimeoutMS' );
+const startBrowserTimeoutMS = config.get( 'startBrowserTimeoutMS' );
+const screenSize = driverManager.currentScreenSize();
+
+var driver;
+
+test.before( function() {
+	this.timeout( startBrowserTimeoutMS );
+	driver = driverManager.startBrowser();
+} );
+
+test.describe( `Jetpack Connect: (${ screenSize }) @jetpack`, function() {
+	this.timeout( mochaTimeOut );
+
+	test.describe( 'Connect From Calypso:', function() {
+		this.bailSuite( true );
+
+		test.before( function() {
+			return driverManager.clearCookiesAndDeleteLocalStorage( driver );
+		} );
+
+		test.it( 'Can create wporg site', () => {
+			this.wporgCreator = new WporgCreatorPage( driver );
+			this.wporgCreator.waitForWpadmin();
```

Comment:
I'm not totally clear about tests like this. There's no return and it's not async via `done`.

Does the test runner wait for `waitForWpadmin`?

Should we be testing some sort of result, or does a lack of errors satisfy the test?

Reply:
> does a lack of errors satisfy the test?

I think so. Breaking things up into separate tests aids debugging. See for example https://github.com/Automattic/wp-e2e-tests/blob/master/scripts/jetpack/wp-jetpack-activate.js